store/src/java/com/zimbra/cs/service/admin/AdminDocumentHandler.java

```diff
@@ -588,10 +588,8 @@ protected Element proxyIfNecessary(Element request, Map<String, Object> context)
             if (acctId != null) {
                 Account acct = getAccount(prov, AccountBy.id, acctId, zsc.getAuthToken());
                 if (acct != null && !Provisioning.onLocalServer(acct, reasons)) {
-                    if (zsc.getHopCount() > 2 || (ZimbraLog.soap.isDebugEnabled())) {
-                        ZimbraLog.soap.info("Proxying request:ProxiedAccountPath=%s reasons:%s",
-                                Joiner.on("/").join(xpath), reasons.getReason());
-                    }
+                    ZimbraLog.soap.info("Proxying request:ProxiedAccountPath=%s reasons:%s",
```

Comment:
Is it worth including the account ID too?  Not sure if it can be worked out by the rest of the details but if not might be useful.  Actually, looking at your manual testing example, looks like reasons contains the account name.  So up to you whether you want to do this.

Reply:
+1 for ```id=%s``` being added (would at least seem to be more consistent), but I could see it potentially being useful too

Reply:
Perhaps the account name is good enough for the operation tracking purpose.  In fact, usually the INFO or higher event does not contain an account ID (Grepping my mailbox.log, I saw only it in the "mbxmgr - Mailbox 3 account <account ID> LOADED").